src/restart.js

```diff
@@ -1,4 +1,22 @@
-module.exports = function restart(req, res) {
-  // TODO
-  res.status(501).send('501 Not Implemented');
+let adb = require('./adb');
+let parse = require('url').parse;
+
+module.exports = async function restart(req, res) {
+  let params = parse(req.originalUrl, true /* parseQueryString */).query;
+  let hard;
```

Comment:
Sorry, that would be URL parsing, just blabbing. :smile: 

But in that instance, using parse may be redundant, since you could just:

```js
let { hard } = JSON.parse(req.query);
```